hepcrawl/utils.py

```diff
@@ -238,9 +239,9 @@ def range_as_string(data):
     ranges = []
     for key, group in groupby(
         enumerate(data),
-        lambda (index, item): index - item
+            lambda index_item: index_item[0] - index_item[1]
     ):
-        group = map(itemgetter(1), group)
+        group = list(map(itemgetter(1), group))
```

Comment:
```suggestion
        group = [year for (_, year) in group]
```
(not that it matters too much, I have no idea why such a function doing formatting would be needed in a software that *extracts* metadata).

Reply:
I agree that comprehensions are more readable than `list(map(  ))`  constructs. 
I should have taken the opportunity to re-write this and other similar expressions
I think `[g[1] for g in group]` is better, because it doesn't rely on group having exactly 2 values to unpack